src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int color = new Random().nextInt(Color.values().length);
+        return Color.values()[color];
     }
```

Comment:
You should avoid creating several instances of the same class if you can use only one instance for your purposes. In this case, you can have a single `Random` instance at the class level.